server/src/server.ts

```diff
@@ -82,6 +84,29 @@ export class Server {
     return sock;
   }
 
+  private updateTemplate(template: Template): Template {
+    template.hash = crypto.createHash("sha256")
+      .update(JSON.stringify(template))
+      .digest("hex");
+
+    const cb0 = "03";                                       // push
```

Comment:
Hmmm, I don't know. Because the `03` represent the `push` command. The comment say what the number is. You would prefer this comment to explain what the push do?